spring-aop/src/main/java/org/springframework/aop/interceptor/AsyncExecutionAspectSupport.java

```diff
@@ -152,7 +160,10 @@ public void setExceptionHandler(AsyncUncaughtExceptionHandler exceptionHandler)
 	 */
 	@Override
 	public void setBeanFactory(BeanFactory beanFactory) {
-		this.beanFactory = beanFactory;
+		if (!Objects.equals( this.beanFactory, beanFactory )) {
```

Comment:
is this passing the style checks?

Reply:
Task :spring-aop:checkstyleMain reports no error

Reply:
In any case, please remove empty space after an opening parenthesis and before a closing parenthesis, since that is the standard convention within the Spring Framework codebase.

And please make that change consistently within this PR.